src/compiler/compile/render_dom/wrappers/Element/Attribute.ts

```diff
@@ -41,9 +41,13 @@ export class BaseAttributeWrapper {
 		}
 	}
 
-	render(_block: Block) {}
+	render(_block: Block) { }
 }
 
+const regex_minus_sign = /-/;
+const regex_invalid_variable_identifier_characters = /[^a-zA-Z_$]/g;
```

Comment:
there are some similar regex patterns
https://github.com/ivanhofer/svelte/blob/more-performant-regex/src/compiler/compile/render_dom/wrappers/Element/index.ts#L144
https://github.com/ivanhofer/svelte/blob/more-performant-regex/src/compiler/compile/render_dom/wrappers/shared/add_actions.ts#L15

But this one is missing the `0-9` part. Should it be added?

Reply:
yup sounds like a good idea